packages/wasm/crate/tests/test_keys.rs

```diff
@@ -87,3 +87,39 @@ fn returns_false_on_unknown_addr() {
             .0;
     assert!(!is_controlled_address(&fvk.encode_to_vec(), &other_address.encode_to_vec()).unwrap());
 }
+
+#[test]
```

Comment:
what about additional test coverage for:

1. edge cases near the sequence range boundaries (eg. check addresses are different for sequence numbers 0 and 2^16 - 1)

2. same sequence and account generate the same address

3. different full viewing keys yield different addresses

Reply:
Adding those additional cases 👍